src/tests/test_utils/common_test_utils/src/postgres_link.cpp

```diff
@@ -557,12 +557,12 @@ class PostgreSQLEventListener : public ::testing::EmptyTestEventListener {
              << ", run_count=" << test_suite.test_to_run_count() << ", total_count=" << test_suite.total_test_count()
              << " WHERE sr_id=" << this->testSuiteId;
         auto pgresult = connectionKeeper->query(sstr.str().c_str(), PGRES_COMMAND_OK);
-        CHECK_PGRESULT(pgresult, "Cannot update test suite results", return);
+        CHECK_PGRESULT(pgresult, "Cannot update test suite results", return );
         this->testSuiteId = 0;
 
         if (reportingLevel == REPORT_LVL_FAST) {
             pgresult = connectionKeeper->query(joinedQuery.str().c_str());
-            CHECK_PGRESULT(pgresult, "Cannot update test cases results", return);
+            CHECK_PGRESULT(pgresult, "Cannot update test cases results", return );
```

Comment:
Why? It's not aligned with code style

Reply:
clang-format changed it automatically. reverted